src/transformers/modeling_gguf_pytorch_utils.py

```diff
@@ -129,6 +129,9 @@ def load_gguf_checkpoint(gguf_checkpoint_path, return_tensors=False):
             )
         model_size = m.group().strip("-")  # only keeps `7b`
 
+    if "nemotron" in architecture:
```

Comment:
Why you explicitly assign architecture to updated one if it is the same?

Reply:
I'm sorry if my answer seems obvious, but isn't it for addressing cases where the "architecture" does not only contain "nemotron"? I took reference on what you did for the qwen2moe, so I think It's better to also do it for nemotron. But I tested it without these lines and it passes through. What do you think? And thank you for reviewing! As this is my first time contributing, please let me know if anything seems odds or is there any better implementation. Thank you!

Reply:
No, for qwen2moe, I explicitly assigned another architecture name, because gguf file contains qwen2moe, but later, execution chains expects to get **qwen2_moe** for processing. You provided the same name "nemotron". So, there is no reason to explicitly assign updated architecture to **the same name**.

Reply:
I'm sorry for misreading the code and thank you for pointing it out! I have deleted the unnecessary lines in the new commit. Please let me know if there is something needs to be fixed.

Reply:
No worries, other changes is fine from my perspective

Reply:
Thank you for reviewing!